### PR TITLE
Vendor trends

### PR DESCRIPTION
**Pull request by volkanbulca**
Can you please review searchtrendforvendor.js file ?

### CODE REVIEW

File: practice-app/back-end/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "dependencies": {
     "cookie-parser": "~1.4.4",
     "debug": "~2.6.9",
-    "express": "^4.16.4",
+    "express": "^4.17.1",
```
Comment (Physicliar) [inline practice-app/back-end/package.json:11]: Did you realized that you updated the express package? Some functionalities may depreciated or changed. It is better to check before merge the branch

Comment (volkanbulca) [reply practice-app/back-end/package.json]: Thank yoı, fixed it.

**Comment by kayacanv** [comment]
@volkanbulca key.js contains actual keys :D

**Comment by sertayy** [comment]
Be sure that you don't have any conflicts with the main branch to merge the endpoint. I guess you need to check the package-lock.json and keys.js files.

**Pull request merged**